aries-core/src/js/components/core/index.js

```diff
@@ -1,16 +1,6 @@
-export * from './Accordion';
-export * from './Button';
-export * from './Checkbox';
 export * from './Form';
-export * from './MaskedInput';
 export * from './Menu';
 export * from './Nav';
-export * from './RadioButtonGroup';
-export * from './RangeInput';
 export * from './RangeSelector';
```

Comment:
I think the RangeSelector is slightly modified (not 100% pass through from grommet), but could be 100% pass-through if the color specified gets moved to the theme instead.